pkg/splunk/enterprise/configuration.go

```diff
@@ -1888,19 +1888,19 @@ func validateLivenessProbe(ctx context.Context, cr splcommon.MetaObject, livenes
 	}
 
 	if livenessProbe.InitialDelaySeconds < livenessProbeDefaultDelaySec {
-		scopedLog.Error(err, "InitialDelaySeconds is too small", "configured", livenessProbe.InitialDelaySeconds, "recommended minimum", livenessProbeDefaultDelaySec)
+		scopedLog.Info("Liveness Probe: Configured  InitialDelaySeconds is too small, recommended default minimum will be used", "configured", livenessProbe.InitialDelaySeconds, "recommended minimum", livenessProbeDefaultDelaySec)
```

Comment:
I am thinking, instead of making them info, how about fixing it this way:
```
if livenessProbe.InitialDelaySeconds !=0 && livenessProbe.InitialDelaySeconds < livenessProbeDefaultDelaySec {
```